modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/show_caps_lock.js

```diff
@@ -8,9 +8,10 @@ export default function showCapsLock(event, elementId) {
 
 	if (element) {
 		element.style.display = 'none';
-
+		element.classList.add('hide');
```

Comment:
Hey @jgarcialr, please add a line break before the if statement here and sort the calls (`element.classList` and `element.style`) if possible.

Thx

Reply:
yes I think this is possible, the sort shouldn't affect the behavior